Assets/Scripts/Player/PlayerControls.cs

```diff
@@ -58,7 +58,7 @@ void Awake()
 		DayCycleManager.RegisterEndOfDayCallback( Respawn );
 	}
 
-	void Update()
+	void FixedUpdate()
 	{

```

Comment:
This fixes the problem of one click giving multiple items. It might also help with some of the jumping problems, but I'm not sure. When we were reading input in `Update()` input was being read less often than physics was being updated in `FixedUpdate()` so 2 or 3 frames would be using the same input data. This fixes it for now, though we should seek out a better solution at some point.